Automattic/wp-calypso
# Change Google My Business Sharing Image

## Specs

Change the Sharing image for Google My Business

### Before
<img width="727" alt="screen shot 2018-06-20 at 5 06 08 pm" src="https://user-images.githubusercontent.com/2810519/41690863-5f0834d6-74ac-11e8-8928-5d365abc68e4.png">

### After
<img width="727" alt="screen shot 2018-06-20 at 5 05 42 pm" src="https://user-images.githubusercontent.com/2810519/41690865-606506b0-74ac-11e8-83a2-7e58aac65b84.png">

## Testing
1. Navigate to `sharing/:site` on a site unconnected to Google My Business
2. Open the Google My Business Sharing Tab
3. Confirm that the sharing image matches the "After" image above.
